src/test/java/tests/LoginTest.java

```diff
@@ -1,22 +1,27 @@
 package tests;
 
+import io.qameta.allure.Feature;
 import org.testng.Assert;
+import org.testng.annotations.Ignore;
 import org.testng.annotations.Test;
 import pages.MainPage;
 import pages.RegistrationPage;
 import util.RandomSymbolUtil;
 
 public class LoginTest extends BaseTest {
 
-    @Test(description = "Test to check if the entrance form opens")
+    @Feature("Sign in/up")
```

Comment:
Why don't you define feature on the class?

Reply:
fixed